examples/server/server.cpp

```diff
@@ -1404,7 +1415,7 @@ struct llama_server_context
             case TASK_TYPE_NEXT_RESPONSE: {
                 // do nothing
             } break;
-            case TASK_TYPE_SLOTS_DATA: {
+            case TASK_TYPE_METRICS: {
```

Comment:
Good idea! when I work in this code base for the first time, I was confused that `TASK_TYPE_SLOTS_DATA` meaning it read or mutate with slot data..